parts/linux/cloud-init/artifacts/components.json

```diff
@@ -42,21 +42,14 @@
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-cni:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.5.28",
-        "v1.6.0"
```

Comment:
why remove 1.6.0?

Reply:
Oh didn't see this. I didn't delete them intentionally. Let me add all back.
Thanks for catching this.